.github/workflows/main.yml

```diff
@@ -12,9 +12,8 @@ jobs:
     strategy:
       matrix:
         config:
-          - ["Py3.8-Plone5.2", "3.8", "5.2"]
-          - ["Py3.8-Plone6.0", "3.8", "6.0"]
-            #          - ["Py3.11-Plone6.0", "3.11", "6.0"]
+          - ["Py3.11-Plone6.0", "3.11", "6.0"]
+          - ["Py3.12-Plone6.0", "3.12", "6.0"]
```

Comment:
Further in this file I see two conditions with `if: ${{ matrix.config[2] == '5.2' }}`.  The conditions can be removed/updated.
Probably `bin/code-analysis` no longer needs to be called, and `pre-commit` is enough.

Reply:
Thanks for spotting this!

Reply:
Change approved.